docs/search/README.md

```diff
@@ -20,15 +16,13 @@ We're taking a [data informed](https://stacks.wellcomecollection.org/data-inform
 
 In broad terms, we want to produce a number which tells us whether a candidate change to the search system makes the results more or less relevant to the person searching.
 
-There's a lot of interesting discussion to be had about different search modalities, precision vs serendipity, linked open data & knowledge graphs, and existential questions about what relevance even _is_, dude, but the broad definition above is a useful, concise statement to revert back to.
```

Comment:
Haha - I wonder how this got here.

Reply:
Poetic licence?

Reply:
Maybe we should try write all docs in rhyming couplets?